salt/states/netconfig.py

```diff
@@ -230,7 +231,7 @@ def replace_pattern(
     ret = salt.utils.napalm.default_ret(name)
     # the user can override the flags the equivalent CLI args
     # which have higher precedence
-    test = __salt__["config.merge"]("test", test)
+    test = test = test or __opts__["test"]
```

Comment:
`test = test = test or` ?

Reply:
sigh.. copy pasta. Not my best day, I will fix

Reply:
should be resolved